eslint.config.mjs

```diff
@@ -36,7 +37,7 @@ Module._resolveFilename = (request, parent, isMain, options) => {
   return ModuleResolveFilename(request, parent, isMain, options);
 };
 
-export default [
+export default tseslint.config(
```

Comment:
Nit: This is optional. We don't need it.

Reply:
It's optional, but it's recommended:
> We strongly recommend using this utility to improve the config authoring experience — however it is entirely optional. By choosing not to use it you lose editor autocomplete and type checking for config files but otherwise it will not impact your ability to use our tooling.

I'm happy to remove it.

Reply:
+1 to remove it